Cargo.toml

```diff
@@ -24,6 +24,8 @@ cortex-m-rt = "0.6"
 embedded-hal = "0.2"
 nb = "0.1"
 stm32f3 = "0.10"
+typenum = "1.11.2"
+embedded-hal-can = "0.1.0"
```

Comment:
Hm. As this is an experimental crate until this is added to the `embedded-hal` we should feature gate the CAN implementation as a whole?

But I'm happy to include it into this hal, so that more people can experiment with the can abstraction! :)

Reply:
ya my plan was to put this whole thing behind a feature flag once it was more merge-ready, but intellij doesn't totally know how to handle feature flags ATM so I have avoided that for ease of development.